misc_python/jobparser.py

```diff
@@ -234,14 +384,53 @@ def _get_job_info(self):
         lines.append("Working Directory: {0}".format(self.WorkingDirectory))
         lines.append("User: {0}".format(self.User))
         lines.append("Comment: {0}".format(self.Comment))
-        lines.append("Scheduled Date: {0}".format(self.ScheduledDate))
+#        lines.append("User Data Size: {0}".format(self.UserDataSize))
+#        lines.append("Reserved Data Size: {0}".format(self.ReservedDataSize))
+        for e in exitcode:
+            if self.StartError == e:
+                serror = exitcode[e]
+        lines.append("Start Error: {0}".format(serror))
+#        lines.append("Task Flags: {0}".format(self.TaskFlags))
+        lines.appensd("Trigger Count: {0}".format(self.TriggerCount))
+        lines.append("Scheduled Start Date: {0}".format(self.ScheduledStart))
+        lines.append("Scheduled End Date: {0}".format(self.ScheduledEnd))
+        lines.append("Start Hour: {0}".format(self.StartHour))
+        lines.append("Start Minute: {0}".format(self.StartMinute))
+        lines.append("Minutes Duration: {0}".format(self.MinutesDuration))
+        lines.append("Minutes Interval: {0}".format(self.MinutesInterval))
+        tflags = ""
+        for flag in triggerflags:
+            if self.TriggerFlag & flag == flag:
+                tflags += triggerflags[flag] + ","
+        lines.append("Trigger Flags: {0}".format(tflags.rstrip(", ")))
+        ttype = ""
+        for type in triggertype:
+            if self.TriggerType & type == type:
+                ttype += triggertype[type] + ","
+        lines.append("Trigger Type: {0}\n".format(ttype.rstrip(", ")))
+        if self.TriggerSpecific != "":
+            lines.append("{0}".format(self.TriggerSpecific))
+#        lines.append("Padding: {0}".format(self.Padding))
```

Comment:
Could you remove any comment out code ?  Or is there a reason to keep it ?

Reply:
Code removed.